File: 100 Days Coding Part-1/25. Area of Circle/areaofcircle.py
#                          Day 25 coding Statement 

# Write a Python program to find Area of a circle

# Description
#  Get the value for radius from the user and calculate the area of the circle for the given radius.
#  Area of circle = pi * radius * radius = 3.14 * radius * radius

# Input
#  Enter radius of circle: 5
# Output
#  Area of circle: 78.5

# Method 1

# radius = float(input("Enter radius of circle: "))
# area = 3.14 * radius * radius
# print("Area of circle: ",area)


# Method 2

from math import pi
radius = float(input("Enter radius of circle: "))
area = pi * radius * radius
print("Area of circle: ",area)

File: 100 Days Coding Part-1/25. Area of Circle/areaofcircle.java
//                                   Day 25 coding Statement 

// Write a Java  program to find Area of a circle

// Description
//  Get the value for radius from the user and calculate the area of the circle for the given radius.
//  Area of circle = pi * radius * radius = 3.14 * radius * radius

// Input
//  Enter radius of circle: 3
// Output
//  Area of circle: 28.26

import java.util.Scanner;
public class areaofcircle
 {
    public static void main(String[] args)
    {
        Scanner sc = new Scanner(System.in);
        System.out.print("Enter radius of Circle: ");
        float radius = sc.nextFloat();
        double area = 3.14 * radius * radius;
        System.out.print("Area of Circle is: "+area);
    }
}

















